scripts/database/README_upgrade_across_versions.txt

```diff
@@ -0,0 +1,64 @@
+We now offer an *EXPERIMENTAL* upgrade method allowing users to skip
+over a number of releases. E.g., it should be possible now to upgrade
+a Dataverse database from v4.8.6 directly to v4.10, without having to
+deploy the war files for the 5 releases between these 2 versions and
+to run the corresponding database upgrade scripts manually.
+
+One more time, it is *EXPERIMENTAL*! DO NOT attempt to run this script
+on your production database WITHOUT BACKING IT UP first!
+
+The script, upgrade.sh, must be run in this directory, as follows:
+
+./upgrade.sh [VERSION_1] [VERSION_2] [PG_HOST] [PG_PORT] [PG_DB] [PG_USER]
```

Comment:
How will users get upgrade.sh? By cloning the git repo for now? Some day it might be nice to make it downloadable from the guides like we do with clear_timer.sh from http://guides.dataverse.org/en/4.9.4/admin/troubleshooting.html or ec2-create-instance.sh from http://guides.dataverse.org/en/4.9.4/developers/deployment.html

Reply:
I _think_ you can use a github raw url for direct downloads.

Reply:
The target audience for this script definitely is *advanced* Dataverse admins. Let's assume that figuring out how to obtain the script is an "exercise for the reader", purposefully left there in order to weed out those who have no business trying to run this script. :)